package.json

```diff
@@ -3,7 +3,7 @@
   "author": "NuoDB, Inc.",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "Apache-2.0",
-  "version": "3.5.1",
+  "version": "3.5.2",
```

Comment:
I still don't understand why the patch version is getting bumped. This is master. The typical tagging convention for releases is to do patch releases off of the dev branch for that major.minor version, and all releases off of master are x.y.0 releases, which the x.y dev branch branches off of.

See the picture below for an example of what I'm talking about:

```
master
o v3.6.0 
|
|  v3.5-dev
|  o v3.5.1
| /
|/
o v3.5.0
|
|      v3.4-dev
|      o v3.4.2
|     /
|    /
|   o v3.4.1
|  /
| /
|/
o v3.4.0
.
.
.
```